scaling_performance/network_optimization.adoc

```diff
@@ -170,6 +170,31 @@ network infrastructure.
 VXLAN-offload does not reduce latency. However, CPU utilization is reduced even
 in latency tests.
 
+[[scaling-performance-optimizing-mtu]]
+=== Optimizing the MTU for Your Network
+
+There are two MTUs (maximum transmission units) that matter: the network interface card (NIC) MTU, and the SDN overlay's MTU.
+
+The NIC MTU can be whatever you think is optimal for your network.
```

Comment:
"The NIC MTU can be whatever you think is optimal for your network."

You can't just pick a number.  It must be lss than or equal to the maximum that all of your NICs on the network support.  But you can pick a lower number for the reasons Jeremy outlined.  Optimizing for throughput?  Pick the largest possible. Optimizing for lowest latency?  Pick something lower.